msteams-platform/graph-api/meeting-transcripts/overview-transcripts.md

```diff
@@ -1,54 +1,56 @@
+| You need to obtain transcripts and recordings for capturing meaningful insights from multiple meetings across the Sales vertical. It's time-consuming and inefficient to keep track of all meetings, and to retrieve meeting transcripts and recordings manually. After the meeting is over, you'd need to examine conversations in all those meetings to obtain useful information. | Using Graph APIs in your app to fetch meeting transcripts and recordings automatically retrieves them from all meetings relevant for your purpose. Your app can receive meeting notifications, and get the transcript and recording when it's generated after the meeting ends. This data can then be used to gain: <br> • Aggregated insights and intelligence analysis <br> • New leads and highlights <br> • Meeting follow-ups and summaries |
+| As an HR initiative, you're holding a brainstorming session to understand and improve employee health and productivity. Having to continually take notes to provide post-meeting summary can impede the flow of thoughts, and you might not capture all valuable suggestions. After the session, you'd need to analyze the discussion to gather data points for planning improvements. | Using Graph APIs in your app to fetch transcripts and recordings post-meeting frees you and the participants to fully focus on the discussion. The content of the meeting transcript and recording is available for: <br> • Engagement and sentiment analysis <br> • Listing tasks or issues <br> • Follow-up meetings and notifications |
+
+You can fetch meeting transcript, recording, or both using Graph APIs.
 
 > [!NOTE]
 > In the future, Microsoft may require you or your customers to pay additional fees based on the amount of data accessed through the API.
 
-To fetch the transcript for a particular meeting:
+To fetch the transcript and recording for a particular meeting:
 
-- [Configure permissions on Azure AD to access transcript](#configure-permissions-on-azure-ad-to-access-transcript)
+- [Configure permissions on Azure AD to access transcript and recording](#configure-permissions-on-azure-ad-to-access-transcript-and-recording)
 - [Obtain meeting ID and organizer ID](fetch-id.md)
+- [Get notified when transcripts is available](/graph/transcripts)
 - [Use Graph APIs to fetch transcript](/graph/api/resources/calltranscript)
```

Comment:
Same as above, should recording be added here?

Reply:
updated as suggested